packages/volto-slate/src/editor/render.jsx

```diff
@@ -174,20 +174,20 @@ export const renderLinkElement = (tagName) => {
       appPathname.concat(`#${slug}`),
     );
     const intl = useIntl();
-
     return slate.useLinkedHeadings === false ? (
-      <Tag {...attributes} className={className}>
+      <Tag {...attributes} className={className} tabIndex={0}>
         {children}
       </Tag>
     ) : (
-      <Tag {...attributes} className={className}>
+      <Tag {...attributes} className={className} tabIndex={0}>
         {children}
         {mode === 'view' && slug && (
           <UniversalLink
             className="anchor"
             aria-hidden="true"
             tabIndex={-1}
             href={`#${slug}`}
+            aria-label="anchor"
```

Comment:
Is it meaningful to use `aria-label` on an element that also has `aria-hidden="true"`?